broker/aws/impl.go

```diff
@@ -87,15 +87,22 @@ func (b *Broker) withProfileAssumeRole(accountName string, profileName string, r
 	durationSeconds := int64(3600)
 	accountID, err := b.accountIDFromName(accountName)
 	if err != nil {
-		return nil, err
+		return nil, "", err
+	}
+
+	arnRolePrefix := "arn:aws:iam"
+	if region == "us-gov-west-1" {
```

Comment:
Please change this test to: `if strings.HasPrefix(region, "us-gov-")` so that other GovCloud regions are supported.